ethpm_cli/main.py

```diff
@@ -91,6 +92,15 @@ def parse_arguments() -> argparse.ArgumentParser:
         action="store_true",
         help="Flag to use locally running IPFS node.",
     )
+
+    list_parser = subparsers.add_parser("list", help="List installed packages")
+    list_parser.add_argument(
```

Comment:
I would recommend moving the definition of your parser out of this function body either into the top level of this module or even into it's own module to 1. reduce the size of this function body and 2. maybe reduce the size of this module.